src/message_reader.cc

```diff
@@ -99,20 +99,19 @@ std::unique_ptr<pbio::ZeroCopyInputStream> MessageReader::NextMessage() {
     }
 
     // Try to read the delimiter
-    unsigned char delimiter[kDelimiterSize] = {0};
-    if (!ReadStream(in_, delimiter, sizeof(delimiter))) {
+    if (!ReadStream(in_, delimiter_, kGrpcDelimiterByteSize)) {
```

Comment:
Need to reset delimiter_ buffer for each frame,  it could be multiple frames.

Reply:
Hmm, I don't see the need to reset the buffer (requires another `memcpy` to the heap). If there is not enough data to read, the method will either return `nullptr` or bad status and the buffer will never be read. If there are 5 bytes available, all 5 bytes will override the previous non-zero values.

Reply:
If MessageReader is used to handle streaming,  multiple message,  I think we need to reset the delimitor buffer here.

If multiple message is not supported,  please add a note in the class comment.

Reply:
Multiple messages are supported. Done, added `memset` here.